cmd/snap/cmd_login.go

```diff
@@ -113,6 +113,7 @@ func (x *cmdLogin) Execute(args []string) error {
 
 	email := x.Positional.Email
 	if email == "" {
+		fmt.Fprint(Stdout, i18n.G("Authenticated operation on this device will be under terms of service found at: https://www.ubuntu.com/legal/<some-snap-store-url>")+"\n\n")
```

Comment:
I'd recommend moving the URL to somewhere under snapcraft.io

Reply:
why only prompt when email wasn't given?

Reply:
@sparkiegeek who operates snapcraft.io?